src/main/java/com/spartronics4915/frc2020/commands/Spin4TimesCommand.java

```diff
@@ -0,0 +1,52 @@
+// to stop adding it:
+// git reset src/main/java/com/spartronics4915/frc2020/commands/Spin4TimesCommand.java
+package com.spartronics4915.frc2020.commands;
+
+import com.spartronics4915.frc2020.subsystems.PanelRotator;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Spin4TimesCommand extends CommandBase {
```

Comment:
This command should only spin the wheel once. The mechanism controller will click the button for the number of wheel rotations (ex. 4). The reason behind this is that if another team (or us!) messes up by driving away from the wheel too soon, we wouldn't be able to still score it.

Reply:
@j-james I've changed to have it end at 1 spin.